llvm/lib/MC/MCContext.cpp

```diff
@@ -491,28 +491,34 @@ MCSectionMachO *MCContext::getMachOSection(StringRef Segment, StringRef Section,
   return Ret;
 }
 
-MCSectionELF *MCContext::createELFSectionImpl(StringRef Section, unsigned Type,
-                                              unsigned Flags,
-                                              unsigned EntrySize,
-                                              const MCSymbolELF *Group,
-                                              bool Comdat, unsigned UniqueID,
-                                              const MCSymbolELF *LinkedToSym) {
-  MCSymbolELF *R;
-  MCSymbolTableEntry &SymEntry = getSymbolTableEntry(Section);
+template <typename Symbol>
+Symbol *MCContext::getOrCreateSectionSymbol(StringRef Section) {
```

Comment:
Move up to other getOrCreateSymbolXXX()?

Reply:
Thx. Moved